### DISCUSSION TITLE
Accessing recorded videos

### DISCUSSION THREAD

**Discussion by gl2007** [roleoroleo/yi-hack-Allwinner-v2#446] [Q&A] [answered]
Hey all, just started using this hack. Maybe I missed any config pages but I couldn’t find any documentation on accessing the recorded video files. Is FTP the only option otherwise?

Also, how do I get notified of any motions detected? Don’t see any email config option. I will be away for sometime and want to be notified of anyone or anything lurking around my house :)

**Comment by Staini** [comment, marked as answer]
Records are here:
http://camip:8080/?page=eventsdir

And check the home assistant integration:
https://github.com/roleoroleo/yi-hack_ha_integration

**Reply by gl2007** [reply]
Thanks!
